### PR TITLE
add python3-brotli_1.0.9 in meta/recipes-devtools/python

### PR DESCRIPTION
**Pull request by cp0613**
python3-brotli does not support riscv in pypi, so I added it.

Signed-off-by: cp0613 <cp0613@linux.alibaba.com>

### CODE REVIEW

**Comment by QSchulz** [comment]
Hi,

Thank your for your patch, however we do not support GitHub pull requests, this is a mirror just for convenience. Please follow the submission process depicted here: https://www.openembedded.org/wiki/How_to_submit_a_patch_to_OpenEmbedded. Let us know if you have any question.

Cheers

**Comment by cp0613** [comment]
> Hi,
> 
> Thank your for your patch, however we do not support GitHub pull requests, this is a mirror just for convenience. Please follow the submission process depicted here: https://www.openembedded.org/wiki/How_to_submit_a_patch_to_OpenEmbedded. Let us know if you have any question.
> 
> Cheers

OK，Thanks

File: meta/recipes-devtools/python/python3-brotli_1.0.9.bb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Brotli compression format"
+SECTION = "devel/python"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/brotli-${PV}/LICENSE;md5=941ee9cd1609382f946352712a319b4b"
+
+# pypi package does not have a valid license file
+SRC_URI = "http://codeload.github.com/google/brotli/tar.gz/v${PV};downloadfilename=brotli-${PV}.tar.gz"
```
Comment (witekio-kwe) [inline meta/recipes-devtools/python/python3-brotli_1.0.9.bb:7]: This is not true - [Brotli-1.0.9.zip ](https://files.pythonhosted.org/packages/2a/18/70c32fe9357f3eea18598b23aa9ed29b1711c3001835f7cf99a9818985d0/Brotli-1.0.9.zip) does contain a LICENSE file

File: meta/recipes-devtools/python/python3-brotli_1.0.9.bb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+SRC_URI[md5sum] = "c2274f0c7af8470ad514637c35bcee7d"
```
Comment (witekio-kwe) [inline meta/recipes-devtools/python/python3-brotli_1.0.9.bb:9]: md5sum is outdated - not really needed to add it

File: meta/recipes-devtools/python/python3-brotli_1.0.9.bb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+SRC_URI[sha256sum] = "f9e8d81d0405ba66d181529af42a3354f838c939095ff99930da6aa9cdf6fe46"
+
+S = "${WORKDIR}/brotli-${PV}"
+
+inherit setuptools3
+
+RDEPENDS_${PN} = "\
```
Comment (witekio-kwe) [inline meta/recipes-devtools/python/python3-brotli_1.0.9.bb:16]: That's the old recipe syntax - it should be `RDEPENDS:${PN}`

File: meta/recipes-devtools/python/python3-brotli_1.0.9.bb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Brotli compression format"
+SECTION = "devel/python"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/brotli-${PV}/LICENSE;md5=941ee9cd1609382f946352712a319b4b"
```
Comment (witekio-kwe) [inline meta/recipes-devtools/python/python3-brotli_1.0.9.bb:4]: `${WORKDIR}/brotli-${PV}` is not really needed here, as `${S}` is the base path for searching these files

File: meta/recipes-devtools/python/python3-brotli_1.0.9.bb (hunk 1 above)
Comment (cp0613) [reply meta/recipes-devtools/python/python3-brotli_1.0.9.bb:7]: Thanks for reminding, the commit is wrong

**Pull request closed without merging**